tests/plugins/kernel_install/utils_test.sh

```diff
@@ -72,6 +72,7 @@ function setUp()
 function tearDown()
 {
   rm -rf "$SHUNIT_TMPDIR"
+  mkdir -p "$SHUNIT_TMPDIR"
```

Comment:
I think it makes more sense to move this `mkdir -p` from `terarDown` to `setUp`.